### DISCUSSION TITLE
How to configure Antivirus when disabling clamav?

### DISCUSSION THREAD

**Discussion by Extarys** [nextcloud/all-in-one#5755] [Questions] [answered]
I noticed that the Antivirus configuration disappear when we uncheck ClamAV in the AIO interface. I need to use another AV solution with ICAP but I don't want clamav to run in the background for nothing.

How can I enable the Antivirus config section without enabling ClamAV?

**Comment by Extarys** [comment]
:facepalm: AIO install/remove the Antivirus app automatically using dark magic. :confused: Fixed by going to the App store hehe

**Reply by Extarys** [reply]
RIP this doesn't work, when stopping and starting the containers from AIO dashboard, the application is uninstalled.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-keep-disabled-apps

**Reply by Extarys** [reply]
Oops sorry and thanks ! I missed that section. <3
